src/Console/StdOut.php

```diff
@@ -21,7 +24,7 @@ class StdOut
      */
     public static function enableAnsi(): void
     {
-        static::$ansiEnabled = true;
+        static::$ansiEnabled[ get_called_class() ] = true;
```

Comment:
Can we see if `static::$ansiEnabled[static::class]` works instead of `static::$ansiEnabled[ get_called_class() ]`?